flanker/mime/message/part.py

```diff
@@ -631,7 +631,7 @@ def _encode_charset(preferred_charset, text):
 def _encode_transfer_encoding(encoding, body):
     if six.PY3:
         if encoding == 'quoted-printable':
-            body = quopri.encodestring(body, quotetabs=False)
+            body = fix_leading_dot(quopri.encodestring(body, quotetabs=False))
```

Comment:
Could you please break it into two lines. Wrapping one function call into another is harder to debug, because you cannot set a break point in between.